lerna.json

```diff
@@ -2,7 +2,14 @@
   "lerna": "3.16.4",
   "command": {
     "run": {
-      "scope": ["msal", "@azure/msal-common", "@azure/msal-browser", "@azure/msal-angular"]
+      "scope": [
+        "msal", 
+        "@azure/msal-common", 
+        "@azure/msal-browser", 
+        "@azure/msal-angular",
+        "vanilla-js-test-app",
+        "vanilla-js-test-app-2.0"
```

Comment:
Can we confirm msal-node works even with dummy tests? Or is that not a concern?

Reply:
wait! Is this only browser specific end-to-end?

Reply:
I think we can include all end to end tests with this command, I will add that here. For msal-browser and msal-core, the end to end tests are all browser based.

Reply:
You can add whatever sample runs your node end to end tests when you have it created.